server/controllers/api/updateFlight.js

```diff
@@ -1,11 +1,12 @@
 import models from '../../database/models/index';
 
 export default (req, res, next) => {
-  const flightNumber = req.params.flightId;
+  const flightNumber = req.params.flightId || req.body.flightNo;
 
   models.Flights.findOne({ where: { flight_no: flightNumber } }).then((result) => {
     let resultFlightInformation = {};
```

Comment:
@Balsam-Faysal Why not
`const resultFlightInfo = result.dataValues;`?

Reply:
done